C-Plus-Plus/README.md

```diff
@@ -1 +1 @@
-## Add the links to files you have added over here while creating a PR.
\ No newline at end of file
+[Program](https://github.com/sukritishah15/DS-Algo-Point/blob/master/C-Plus-Plus/nth_term_ap.cpp) to find the nth term in the A.P 
```

Comment:
@niranjantuty Please write this as
[Find nth term in an A.P.](C-Plus-Plus/nth_term_ap.cpp)

Rest is Perfect and good to go!
Also, let me know, once you make the above change.

Reply:
@sukritishah15  I have made the required changes.

Reply:
Perfect @niranjantuty
Merging this PR. 
PR's made on or after Oct. 1 shall contribute to Hacktoberfest.
So, stay tuned, we will be adding more issues!

Reply:
Thank you @sukritishah15 for providing opportunity.